README.md

```diff
@@ -1 +1,49 @@
-# GSKHackathon
\ No newline at end of file
+# GSKHackathon
+
+## About
+Apache Cordova is a wrapper framework to allow diverse web technologies and cross-
+platform such as HTML5, JS, and CSS. Execution in the application is required for
+each platform, which relies in API bindings to deploy capable device when needed.
+
+![architecture][logo]
+
+<!-- ![architecture][offlinelogo] -->
+
+[logo]: https://github.com/jraleman/GSKHackathon/blob/zeid/res/cordovaapparchitecture.png
+<!-- [offlinelogo]: /Users/zeidtisnes/Desktop/Hackathong/GSKHackathon/res/cordovaapparchitecture.png -->
+
+## How to Run it
+To run Narwhile on Cordova, install the the tools first. For *iOS* users, go to
+`narwell-cordova` and run the program by `npm run iphone`.
+This will create an emulator for iPhone users.
+
+## Required Modules
+
+- `cordova plugin add cordova-plugin-device`
+- `cordova platform add browser`
+- `cordova platform add ios
+`
+## Dependencies
+
+- [Cordova](https://cordova.apache.org/)
+
+## Tools
+
+* `npm install -g npm`
+* `npm install -g cordova`
+
+## Authors
+
+- [Jose Ramón Aleman](https://github.com/jraleman)
+- [Daniel Gonzales]()
+- [Gerardo Malpartida](https://github.com/regien)
+- [Felipe Romo](https://github.com/felromo)
+- [Zeid Tisnes](https://github.com/zedin27)
+
+## Resources
+* [shitgoeshere (Cordova Apache)](https://cordova.apache.org/docs/en/latest/guide/cli/index.html)
+* [moreshitgoeshere (Platform building)](https://cordova.apache.org/docs/en/latest/reference/cordova-cli/index.html#cordova-platform-command)
```

Comment:
moreshitgoeshere?

Reply:
hahahaha I was bored and i didnt know what to name my links lmao

Reply:
anyways, read the comments of ramon and move `narwell-cordova` in `src/cordova`